ui/components/component-library/button-link/button-link.scss

```diff
@@ -1,42 +1,40 @@
 .mm-button-link {
-  color: var(--color-primary-default);
```

Comment:
Some CSS can now be removed in favour of Box props. Also updating css to use `not` keyword for psuedo state class names